sacc/sacc.py

```diff
@@ -641,6 +641,34 @@ def get_tracer_combinations(self, data_type=None):
         indices = self.indices(data_type=data_type)
         return unique_list(self.data[i].tracers for i in indices)
 
+    def rename_tracer(self, name, new_name):
+        """
+        Get the tracer object with the given name
+
+        Parameters
+        -----------
+        name: str
+            A string name of a tracer to be changed the name
+        new_name: str
+            A string with the new name of the tracer
+
+        """
+
+        tr = self.tracers.pop(name)
+        tr.name = new_name
+        self.tracers[new_name] = tr
+
+        for d in self.data:
+            tr1, tr2 = d.tracers
```

Comment:
The tracers attribute isn't always length 2, it can be anything (e.g. for some things it's just length 1, or bispectra it could be length 3).

Reply:
Good point. Done